src/Xamarin.Android.Build.Tasks/Tasks/Legacy/ValidateJavaVersion.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+using System.Text.RegularExpressions;
+
+namespace Xamarin.Android.Tasks.Legacy
+{
+	/// <summary>
+	/// ValidateJavaVersion's job is to shell out to java and javac to detect their version
+	/// </summary>
+	public class ValidateJavaVersion : Xamarin.Android.Tasks.ValidateJavaVersion
+	{
+		public string AndroidSdkBuildToolsVersion { get; set; }
+
+		public string TargetFrameworkVersion { get; set; }
+
+		protected override bool ValidateJava(string javaExe, Regex versionRegex)
+		{
+			Version requiredJavaForFrameworkVersion = GetJavaVersionForFramework();
+			Version requiredJavaForBuildTools = GetJavaVersionForBuildTools();
+			Version required = requiredJavaForFrameworkVersion > requiredJavaForBuildTools ? requiredJavaForFrameworkVersion : requiredJavaForBuildTools;
+
+			MinimumRequiredJdkVersion = required.ToString();
+
+			try
+			{
+				var versionNumber = GetVersionFromTool(javaExe, versionRegex);
+				if (versionNumber != null)
```

Comment:
Why is this file following a foreign coding convention?

For that matter, should this file be a `git cp` of `src/Xamarin.Android.Build.Tasks/Tasks/ValidateJavaVersion.cs` then with changes applied?

Reply:
Something weird happened, it's like this file got auto-formatted but not using our `.editorconfig`?

Reply:
I don't think there is a `git cp`: https://stackoverflow.com/questions/47401843/git-copy-file-as-opposed-to-git-mv

But I just fixed this file, I don't see others messed up.